TACC/WMA-Tapis-Templates
# task/WA-155: Potree Converter

## Overview: ##
Converts Potree Converter DS app to run in Tapis v3

## PR Status: ##


## Related Jira tickets: ##

* [WA-155](https://tacc-main.atlassian.net/browse/WA-155)

## Summary of Changes: ##
Created a new App Definition and tapisjob_app file to work with Potree converter v2.1.1 and the wma-exec-01 VM

## Testing Steps: ##
_App already tested at DS next_
1. Submit a job at : https://designsafeci-next.tacc.utexas.edu/rw/workspace/potree-converter?appVersion=2.1.1 or [dev.cep](https://dev.cep.tacc.utexas.edu/workbench/applications/potree-converter?appVersion=2.1.1)
2. Confirm job runs successfully and produces the following output:
    - index.html (file)
    - pointclouds (folder) (and is populated)


## UI Photos:

## Notes: ##
